ContentProtection/BasicPlayReady/Program.cs

```diff
@@ -45,7 +44,7 @@ public static async Task Main(string[] args)
 
             }
 
-            ConfigWrapper config = new ConfigWrapper(new ConfigurationBuilder()
+            ConfigWrapper config = new(new ConfigurationBuilder()
```

Comment:
Is that a newer syntax to just do  "new ("

Reply:
Yes, this was suggested by Visual Studio. I find it very cool. I tried to remove all warning and information messages reported by VS.

Reply:
Good with this.